packages/lesswrong/lib/vulcan-lib/routes.ts

```diff
@@ -119,12 +119,13 @@ export const overrideRoute = (...routes: Route[]): void => {
   addRoute(...routes);
 }
 
-export const getRouteMatchingPathname = (pathname: string): Route | undefined  =>
-  Object.values(Routes).findLast((route) => matchPath(pathname, {
+export const getRouteMatchingPathname = (pathname: string): Route | undefined  => {
+  return Object.values(Routes).reverse().find((route) => matchPath(pathname, {
```

Comment:
consider using lodash findLast instead? (nicer and plausibly better for perf, though plausibly perf difference would be miniscule)

Reply:
(I prefer native functionality where available, out of the vain hope that one day we may retire such libraries entirely)